### PR TITLE
change caption on githubteacher's file

### PR DESCRIPTION
**Pull request by gh7166**
This pull request changes this caption

This pull request is in relation to #4 

@brianamarie could you please review ?

This will be complete when...
- [ ] Todo

### CODE REVIEW

File: _posts/0000-01-16-gh7166.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![sad-puppy](https://cloud.githubusercontent.com/assets/16547949/25401096/b9e619ee-29c2-11e7-8ee3-74a301737d34.jpg)
 {: .center}
 
-Caption here
+Hello.
```
Comment (gh7166) [inline _posts/0000-01-16-gh7166.md]: This change looks good.

File: _posts/0000-01-16-gh7166.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![sad-puppy](https://cloud.githubusercontent.com/assets/16547949/25401096/b9e619ee-29c2-11e7-8ee3-74a301737d34.jpg)
 {: .center}
 
-Caption here
```
Comment (gh7166) [inline _posts/0000-01-16-gh7166.md]: 👍

File: _posts/0000-01-16-gh7166.md (hunk 1 above)
Comment (m1a2i3) [reply _posts/0000-01-16-gh7166.md]: good :octocat:

File: _posts/0000-01-16-gh7166.md (hunk 2 above)
Comment (m1a2i3) [reply _posts/0000-01-16-gh7166.md]: ok! 👍

File: _posts/0000-01-16-gh7166.md (hunk 1 above)
Comment (m1a2i3) [reply _posts/0000-01-16-gh7166.md]: 💯

**Pull request merged**